src/root.component.tsx

```diff
@@ -3,11 +3,41 @@ import { Route } from "react-router";
 import { BrowserRouter } from "react-router-dom";
 import openmrsRootDecorator from "@openmrs/react-root-decorator";
 import PatientChartSummary from "./summary/patient-chart-summary.component";
+import PatientBanner from "./summary/banner/patient-banner.component";
+import { getCurrentPatient } from "@openmrs/esm-api";
+import { LevelTwoRoutes } from "./summary/level-two-routes";
 
 function Root(props) {
+  const [currentPatient, setCurrentPatient] = React.useState(null);
+  const [showPatientSummary, setShowPatientSummary] = React.useState(true);
+
+  React.useEffect(() => {
+    const subscription = getCurrentPatient().subscribe(patient => {
+      setCurrentPatient(patient);
+    });
+
+    return () => subscription.unsubscribe();
+  });
   return (
-    <BrowserRouter>
-      <Route to="/patient/:patientUuid/chart" component={PatientChartSummary} />
+    <BrowserRouter basename={window["getOpenmrsSpaBase"]()}>
+      <Route path="/patient/:patientUuid/chart">
+        <PatientBanner
+          match={props.match}
+          patient={currentPatient}
+          showPatientSummary={setShowPatientSummary}
+        />
+      </Route>
+      <Route
+        path="/patient/:patientUuid/chart"
+        exact
+        component={PatientChartSummary}
+      />
+      {currentPatient && (
```

Comment:
@donaldkibet why is `currentPatient` here ? what happens when you remove it ?

Reply:
Agreed that this looks wrong, since we're not even passing `currentPatient` down as a prop to LevelTwoRoutes.